openhtf/io/frontend/__main__.py

```diff
@@ -133,12 +200,21 @@ def main(argv):
     print('%s\nUsage: %s ARGS\n%s' % (e, sys.argv[0], FLAGS))
     sys.exit(1)
 
-  if not os.path.isdir(FLAGS.rundir):
-    print('ERROR: OpenHTF run directory does not exist', FLAGS.rundir,
-          file=sys.stderr)
-    sys.exit(1)
+  logs.setup_logger()
+
+  frontend_server = HTTPServer(WSGIContainer(app))
```

Comment:
Flask's built-in server is only intended as a dev server. Their own documentation advises against using it in prod. Previously we used Rocket as our web server, and using Flask's built-in dev server was a temporary thing while I investigated other options. I think tornado looks like a better overall framework fit for our needs and adding it here is a step towards replacing flask entirely.